.gitignore

```
@@ -1,3 +1,6 @@
 .stack-work
 *.hie
 .hiedb
+stack-*.lock
+stack.*.lock
```

I used `.stack-*.lock` specifically because I wanted to keep `stack.yaml.lock` checked in. The choice isn't particularly important, but you could simplify if excluding it all:
```suggestion
stack*.yaml.lock
```

---

Doesn't matter but it always bothers me to put wildcards right next to words because I wouldn't want to match words that happen to begin with "stack," `stackalackadingdong.yaml.lock` or whatever.